manifests/config.pp

```diff
@@ -1,7 +1,19 @@
 # == Class chocolatey::config
-#
-# This class is called from chocolatey for service config.
-#
-# NOTE: CURRENTLY NON-FUNCTIONAL
 class chocolatey::config {
+  assert_private()
+
+  #todo: check choco version from custom choco_version fact
+
+  $_choco_exe_path = "${chocolatey::choco_install_location}\\bin\\choco.exe"
+
+  $_enable_autouninstaller = $chocolatey::enable_autouninstaller ? {
+    false => 'disable',
+    default => 'enable'
+  }
+
+  exec { "chocolatey_autouninstaller_${_enable_autouninstaller}":
+    path    => $::path,
+    command => "${_choco_exe_path} feature -r ${_enable_autouninstaller} -n autoUninstaller",
+    unless  => "${_choco_exe_path} feature list -r | findstr /X /I \"autoUninstaller.*\\[${_enable_autouninstaller}d\\]\"",
```

Comment:
Attempted to escape but that doesn't work either.

![image](https://cloud.githubusercontent.com/assets/63502/9510506/15c60244-4c32-11e5-9e59-63aae703ae0f.png)